src/main/java/streambms/BatteryManagementSystem.java

```diff
@@ -0,0 +1,59 @@
+package streambms;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import streambms.constant.BatteryConstant;
+import streambms.response.BatteryResponse;
+
+public class BatteryManagementSystem {
+
+	public static BatteryResponse sendBatteryParameters(BatteryResponse battery) {
+		if (battery == null)
+			return null;
+
+		return battery;
+	}
+
+	public static BatteryResponse getBatteryParameters() {
+		float temperature = getRandomNumber(BatteryConstant.MIN_TEMPERATURE, BatteryConstant.MAX_TEMPERATURE);
+		float soc = getRandomNumber(BatteryConstant.MIN_SOC, BatteryConstant.MAX_SOC);
+		float chargeRate = getRandomNumber(BatteryConstant.MIN_CHANGE_RATE, BatteryConstant.MAX_CHANGE_RATE);
+
+		return new BatteryResponse(temperature, soc, chargeRate);
+	}
+
+	public static void sendToReceiver(boolean isConsumed) {
+		ObjectMapper mapper = new ObjectMapper();
+		try {
+			for (int i = 0; i <= 10; i++) {
+				sendToConsole(mapper.writeValueAsString(sendBatteryParameters(getBatteryParameters())), isConsumed);
+			}
+			
+		} catch (JsonProcessingException e) {
```

Comment:
Nice way to catch specific exception locally. See if you can make a better handling, as per the needs of a consumer of this software. When this goes wrong, what does a user need to know?